ts/screens/authentication/SpidCIEInformationScreen.tsx

```diff
@@ -0,0 +1,165 @@
+    alignItems: "center"
+  },
+  tabBarContainer: {
+    elevation: 0
+  },
+  tabBarUnderline: {
+    borderBottomColor: customVariables.brandPrimary,
+    borderBottomWidth: customVariables.tabUnderlineHeight
+  },
+  viewStyle: {
+    padding: customVariables.contentPadding
+  },
+  activeTextStyle: {
+    fontSize: 14,
+    color: customVariables.brandPrimary
+  }
+});
+
+class SpidCIEInformationScreen extends React.Component<Props, never> {
+  private getValueContent(value: string, content: string) {
+    return (
+      <Row style={styles.row}>
+        <Col size={1}>
+          <H2 style={styles.value}>{value}</H2>
+        </Col>
+        <Col size={5}>
+          <Text>{content}</Text>
+        </Col>
+        <Col size={2} />
+      </Row>
+    );
+  }
+
+  private browseToLink(url: string) {
+    Linking.openURL(url).catch(() => {
+      Toast.show({ text: I18n.t("genericError") });
+    });
+  }
+
+  public render() {
+    return (
+      <BaseScreenComponent goBack={true}>
+        <Container>
+          <View style={styles.viewStyle}>
+            <H2>{I18n.t("authentication.spid_information.contentTitleCie")}</H2>
+
+            <View spacer={true} large={true} />
+            <Text>{I18n.t("authentication.spid_or_cie")}</Text>
+          </View>
+          <Tabs
+            tabBarUnderlineStyle={styles.tabBarUnderline}
+            tabContainerStyle={styles.tabBarContainer}
+          >
+            <Tab heading={I18n.t("authentication.spid")}>
+              <Content>
+                <Markdown>
+                  {I18n.t("authentication.spid_information.spid")}
+                </Markdown>
+                <Grid>
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point1-value"),
+                    I18n.t("authentication.spid_information.point1-content")
+                  )}
+
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point2-value"),
+                    I18n.t("authentication.spid_information.point2-content")
+                  )}
+
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point3-value"),
+                    I18n.t("authentication.spid_information.point3-content")
+                  )}
+
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point4-value"),
+                    I18n.t("authentication.spid_information.point4-content")
+                  )}
+                </Grid>
+                <View spacer={true} extralarge={true} />
+              </Content>
+              <View footer={true}>
+                <Button
+                  block={true}
+                  primary={true}
+                  onPress={() => this.browseToLink("https://www.spid.gov.it")}
+                >
+                  <Text>
+                    {I18n.t("authentication.spid_information.knowMore")}
```

Comment:
It should be "Ottieni SPID" https://projects.invisionapp.com/share/DSUE5POGWHA#/screens